app/models/application.rb

```diff
@@ -11,7 +11,7 @@ class Application < ApplicationRecord
     establishment-ineligible
   ].freeze
 
-  has_paper_trail only: %i[lead_provider_approval_status participant_outcome_state]
+  has_paper_trail meta: { note: :version_note }
```

Comment:
could we add a comment in both application/user model to explain that the created at isn't the original please?

Reply:
@cwrw the version that would get created during migration will include the note here:
https://github.com/DFE-Digital/npq-registration/pull/1848/files#diff-a44bc9ad55c1f2d7565d5a1a513776fb08f5180b4934350ee83fba59bad1d52cR95

So all the changes from the original will be shown. Do you think we need to be more specific?

Reply:
discussed in person: let's just leave in papertrail